backend/src/main/conf/local.json

```diff
@@ -15,5 +31,25 @@
         "window.tolerance.secs": 60
       }
     }
+  },
+  "curatorOptions": {
+    "connection.url": "",
+    "namespace": "fkprof",
+    "connection.timeout.ms": 5000,
```

Comment:
So zookeeper client pings server (with which it has opened the connection) on a regular basis. readTimeout is defined as 2/3rd of session timeout (negotiated).